### PR TITLE
move to poetry

### PR DESCRIPTION
**Pull request by Drugsosos**
Moved to Poetry

It's great, it checks python version, automatically creates venv, locks all dependencies (even from other dependencies) in lock file and lots more

TODO:
---
- [ ] moved to poetry
- [x] updated README.md with new instructions

### CODE REVIEW

**Comment by William9923** [comment]
Hi, if I may ask, where is the readme containing the instruction (?).

And could you also explain what is the reasoning behind requirements.txt -> poetry ? Thanks 🙏

**Comment by Drugsosos** [comment]
> Hi, if I may ask, where is the readme containing the instruction (?).
> 
> 
> 
> And could you also explain what is the reasoning behind requirements.txt -> poetry ? Thanks 🙏 

1) there are commands in README to install dependencies and run script, I updated them, you can check changed files
2) it's just better, it checks python version, creates venv automatically etc

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+[tool.poetry]
+name = "redditvideomakerbot"
+version = "2.2.2"
+description = "Create Reddit Videos with just✨  one command"
+authors = ["Your Name <you@example.com>"]
```
Comment (Drugsosos) [inline pyproject.toml:5]:
```suggestion
description = "Create Reddit Videos with just one command"
authors = ["elebumm <hi@lewismenelaws.com>"]
```

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+[tool.poetry]
+name = "redditvideomakerbot"
+version = "2.2.2"
```
Comment (Drugsosos) [inline pyproject.toml:3]:
```suggestion
version = "2.3.0"
```

File: pyproject.toml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+description = "Create Reddit Videos with just✨  one command"
+authors = ["Your Name <you@example.com>"]
+license = "GPL-3.0 license"
+
+[tool.poetry.dependencies]
+python = ">=3.9,<4"
```
Comment (Drugsosos) [inline pyproject.toml:9]:
```suggestion
python = "^3.9"
```

File: pyproject.toml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+boto3 = "^1.24.24"
```
Comment (Drugsosos) [inline pyproject.toml:10]:
```suggestion
boto3 = "1.24.24"
```

**Comment by JasonLovesDoggo** [comment]
I mean I don't see any changes to the README

**Comment by Drugsosos** [comment]
> I mean I don't see any changes to the README

@JasonLovesDoggo 
sorry, I believe it's chilling in stage 🐸

**Comment by JasonLovesDoggo** [comment]
at the end of the day, this REALLY is unnecessary while for SOME people it might make it a little bit easier. those types of people would probably be the more tech-savvy kind anyways.

**Pull request closed without merging**